docs/nodes/consensus-node.mdx

```diff
@@ -143,53 +209,78 @@ cd $HOME<br/>
 rm -rf ~/.celestia-app/data<br/>
 mkdir -p ~/.celestia-app/data<br/>
 SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \<br/>
-&nbsp; egrep -o ">{constants.mochaChainId}.*tar" | tr -d ">")<br/>
+    egrep -o ">{constants.mochaChainId}.*tar" | tr -d ">")<br/>
 wget -O - https://snaps.qubelabs.io/celestia/$&#123;SNAP_NAME&#125; | tar xf - \<br/>
-&nbsp; -C ~/.celestia-app/data/<br/>
+    -C ~/.celestia-app/data/<br/>
 </code></pre>
 
 </TabItem>
-<TabItem value="blockspacerace" label="Blockspace Race">
+<TabItem value="arabica" label="Arabica">
 
-Run the following command to quick-sync from a snapshot for `blockspacerace`:
+Run the following command to quick-sync from a snapshot for `arabica-10`:
 
-```bash
-cd $HOME
-rm -rf ~/.celestia-app/data
-mkdir -p ~/.celestia-app/data
-SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \
-    egrep -o ">blockspacerace.*tar" | tr -d ">")
-wget -O - https://snaps.qubelabs.io/celestia/${SNAP_NAME} | tar xf - \
-    -C ~/.celestia-app/data/
-```
+<pre><code>
+cd $HOME<br/>
+rm -rf ~/.celestia-app/data<br/>
+mkdir -p ~/.celestia-app/data<br/>
+SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \<br/>
+    egrep -o ">{constants.arabicaChainId}.*tar" | tr -d ">")<br/>
+wget -O - https://snaps.qubelabs.io/celestia/$&#123;SNAP_NAME&#125; | tar xf - \<br/>
+    -C ~/.celestia-app/data/<br/>
+</code></pre>
 
 </TabItem>
 </Tabs>
-````
+```
 
-### Start the celestia-app
+### Start the consensus node
 
-In order to start your validator node, run the following:
+In order to start your full consensus node, run the following:
+
+```sh
+celestia-appd start
+```
+
+Follow the tutorial on
+[setting up Celestia App as a background process with SystemD](./systemd.md).
 
 :::tip
 Refer to
 [the ports section of the celestia-node troubleshooting page](../../nodes/celestia-node-troubleshooting/#ports)
 for information on which ports are required to be open on your machine.
 :::
 
-```bash
-celestia-appd start
-```
+## Optional: Setting up a validator
 
-Follow
-[the tutorial on setting up `celestia-app` as a background process
-with SystemD](../systemd).
+### Setting up a Celestia validator node
+
+Validator nodes allow you to participate in consensus in the Celestia network.
+
+![validator node](../img/nodes/validator.png)
 
-### Wallet
+#### Validator hardware requirements
+
+The following hardware minimum requirements are recommended for running the
+validator node:
```

Comment:
```suggestion
The following hardware minimum requirements are recommended for running a
validator node:
```

Reply:
d2cea5356b2a6d07a84b0f778ff47e4a99dc00b8

Reply:
whoops my b i meant to just commit your suggestion